src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -50,9 +50,19 @@ def load_arguments(self, _):
     isolated_sku_arg_type = CLIArgumentType(help='The Isolated pricing tiers, e.g., I1 (Isolated Small), I2 (Isolated Medium), I3 (Isolated Large)',
                                             arg_type=get_enum_type(['I1', 'I2', 'I3']))
 
+    functionapp_runtime_to_version = {}
+    for version in RUNTIME_TO_IMAGE_FUNCTIONAPP.values():
+        for runtime, val in version.items():
+            if runtime in functionapp_runtime_to_version:
+                functionapp_runtime_to_version[runtime] = set(functionapp_runtime_to_version[runtime]).union(set(val.keys()))
+            else:
+                functionapp_runtime_to_version[runtime] = set(val.keys())
+
     functionapp_runtime_to_version_texts = []
-    for runtime, val in RUNTIME_TO_IMAGE_FUNCTIONAPP.items():
-        functionapp_runtime_to_version_texts.append(runtime + ' -> [' + ', '.join(val.keys()) + ']')
+    for runtime, runtime_versions in functionapp_runtime_to_version.items():
+        runtime_versions_list = list(runtime_versions)
+        runtime_versions_list.sort(key=float)
```

Comment:
The set will perform auto sort, we don't need to do this explicitly.
The next line `', '.join` will iterate through the set. So we may not need these two lines.

Reply:
Yeah so this was a bit frustrating because if I just join the set, it'll print out node versions as "8, 12, 10" for some reason. I can't sort the set so if I turn it into a list and sort it, it'll print "10, 12, 8" because 1 comes before 8. Setting the `key` to `float` will sort them as if they're floats and behave how we would want.

Reply:
OK, I see, yea, that makes sense. We can keep them here.